tests/deployment/ipmi.py

```diff
@@ -67,3 +68,10 @@ def test_node_power_off(self):
         nodes = self.collect_nodes()
         for node in nodes:
             api.node_power_off(node.label)
+
+    def test_node_set_bootdev(self):
+        nodes = self.collect_nodes()
+        for node in nodes:
+            api.node_set_bootdev(node.label, 'pxe')
+            api.node_set_bootdev(node.label, 'disk')
+            api.node_set_bootdev(node.label, 'none')
```

Comment:
It might be good to have at least one or two negative tests, checking that proper exceptions are thrown.